packages/@vue/eslint-config-standard/package.json

```diff
@@ -21,9 +21,9 @@
   },
   "homepage": "https://github.com/vuejs/vue-cli/tree/dev/packages/@vue/eslint-config-standard#readme",
   "dependencies": {
-    "eslint-config-standard": "^12.0.0-alpha.0",
-    "eslint-plugin-import": "^2.11.0",
-    "eslint-plugin-node": "^6.0.1",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-node": "^7.0.1",
     "eslint-plugin-promise": "^3.7.0",
```

Comment:
`^4.0.1` ?

Reply:
I'm planning to open a separate PR to update ESLint to make it easier to review & test.